content/en/blog/news/FlyLight_split_release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Larval split-GAL4 line raw images release"
+linkTitle: "Larval split-GAL4 raw images"
+date: 2024-03-19
+description: >
+    Raw images from the 1020 Larval split-GAL4 lines and 350 Generation 1 LexA lines included in Meissner et al., 2024 are now available on VFB.
+---
+
+The raw image data for the larval split-GAL4 and Gen 1 LexA lines from Meissner, G. W., et al. "A split-GAL4 driver line resource for Drosophila CNS cell types." bioRxiv (2024)._ DOI: [10.1101/2024.01.09.574419](https://doi.org/10.1101/2024.01.09.574419) is available on VFB here: https://raw.larval.flylight.virtualflybrain.org/
+
+![Example_larval_split_image](/static/images/larval_em.png)
```

Comment:
Change to https://www.virtualflybrain.org/images/larval_em.png

Reply:
done